test/unix/main.ml

```diff
@@ -332,6 +332,21 @@ module Close = struct
     Alcotest.check_raises "flush after double close with raises Closed"
       I.Closed (fun () -> Index.flush rw)
 
+  let restart_twice () =
+    let reuse_name = Context.fresh_name "empty_index" in
+    let rw = Index.v ~fresh:true ~readonly:false ~log_size:4 reuse_name in
+    let k1, v1 = (Key.v (), Value.v ()) in
+    Index.replace rw k1 v1;
+    Index.close rw;
+    let rw = Index.v ~fresh:true ~readonly:false ~log_size:4 reuse_name in
```

Comment:
Could add an optional `~fresh` parameter to the `Context.t.clone` function to avoid building indexes yourself here.